modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php

```diff
@@ -77,8 +77,8 @@ public function buildForm(array $form, FormStateInterface $form_state, ContactFo
 
     $form['enabled'] = [
       '#type' => 'checkbox',
-      '#title' => $this->t('Enable tracking'),
-      '#description' => $this->t('Set to "Enable tracking" to turn on tracking for this contact form.'),
+      '#title' => $this->t('Capture Data'),
```

Comment:
"Capture Data" is what is used on the Webform integration but the description indicates the connection to Marketo so at a minimum we should do something similar here.

Current Webform description = "If checked, captured form information will be sent to Marketo"